DEV-1/DEV-1/SubstringSearcher.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+
+namespace DEV_1
+{   /// <summary>
+    /// Class can find and display substrings without consecutive repetitive symbols.
+    /// </summary>
+    class SubstringSearcher
+    {
+        private string recievedString;
+        /// <summary>
+        /// The class constructor checks if argument from command line is valid.
+        /// </summary>
+        /// <param name="recievedString">Argument from command line</param>
+        public SubstringSearcher(string recievedString)
+        {
```

Comment:
Check recievedString == null first, than recievedString.Length, otherwise it will be exception